src/frontend/next/src/pages/error.tsx

```diff
@@ -0,0 +1,193 @@
+      top: theme.spacing(24),
+    },
+  },
+  h1: {
+    fontWeight: 'bold',
+    opacity: 0.85,
+    color: theme.palette.grey[100],
+    fontSize: '12rem',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      fontSize: '4rem',
+    },
+    [theme.breakpoints.between('md', 'lg')]: {
+      fontSize: '8rem',
+    },
+    [theme.breakpoints.up('xl')]: {
+      fontSize: '12rem',
+    },
+  },
+  h2: {
+    fontSize: '2rem',
+    color: theme.palette.grey[200],
+    marginTop: '1.75rem',
+    lineHeight: 'inherit',
+    letterSpacing: 'inherit',
+    transition: 'all linear 350ms',
+    [theme.breakpoints.between('xs', 'sm')]: {
+      fontSize: '2rem',
+    },
+    [theme.breakpoints.between('md', 'lg')]: {
+      fontSize: '4rem',
+    },
+    [theme.breakpoints.up('xl')]: {
+      fontSize: '8rem',
+    },
+  },
+  h3: {
+    fontFamily: 'monospace',
+    fontSize: '2rem',
+    padding: theme.spacing(4),
+    color: theme.palette.grey[300],
+    borderRadius: theme.spacing(1),
+  },
+  link: {
+    color: 'white',
+    fontFamily: 'Roboto, sans-serif',
+    textDecoration: 'none',
+    fontSize: '1.5rem',
+    margin: '0 0.5rem 0 0.5rem',
+  },
+  fab: {
+    position: 'relative',
+    fontSize: '1.5rem',
+    color: theme.palette.text.primary,
+    bottom: -45,
+    zIndex: 200,
+    backgroundColor: theme.palette.secondary.light,
+    transition: 'all linear 250ms',
+    '&:hover': {
+      backgroundColor: theme.palette.secondary.dark,
+    },
+    [theme.breakpoints.between('xs', 'sm')]: {
+      right: '-2rem',
+    },
+  },
+  buttonText: {
+    fontSize: '1.5rem',
+    paddingLeft: theme.spacing(1),
+    [theme.breakpoints.between('xs', 'sm')]: {
+      display: 'none',
+    },
+  },
+}));
+
+type ErrorProps = {
+  message: string | null;
+  status: number | null;
+};
+
+function CreateInnerErrorContent({ message, status }: ErrorProps) {
+  const classes = useStyles();
+
+  let errorMessage;
+
+  switch (status) {
+    case 400:
+      errorMessage = 'We did not understand the request!';
+      break;
+    case 401:
+      errorMessage = 'You are not authorized to view this page!';
+      break;
+    case 403:
+      errorMessage = 'Access is not allowed for the requested page!';
+      break;
+    case 404:
+      errorMessage = 'We could not find what you were looking for!';
+      break;
+    case 405:
+      errorMessage = 'Method is not allowed!';
+      break;
+    default:
+      errorMessage = 'Something went wrong!';
+  }
+
+  // If server doesn't send us a custom message, use ones defined above.
+  if (!message) {
+    return (
+      <Typography variant="body1" className={classes.h2}>
+        {errorMessage}
+      </Typography>
+    );
+  }
+  return (
+    <Typography variant="body1" className={classes.h3}>
+      {message}
+    </Typography>
+  );
+}
+
+const ErrorPage = () => {
+  const classes = useStyles();
+  const router = useRouter();
+
+  const baseURL = 'https://telescope.cdot.systems/';
```

Comment:
We need to get this from `config.ts` so we can configure it for local vs. staging vs. prod.